PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java

```diff
@@ -28,18 +29,20 @@ public static void main(String[] args) {
         
         String tweeterName = keyboard.nextLine();
         
-        System.out.println("Nice to meet you " + tweeterName + "!");
+        System.out.println("Nice to meet you, " + tweeterName + "!");
         System.out.println("Enter your tweets and I will add them to your timeline!");
         
         int numTweets = 0;
         
         while(numTweets < (MAX_NUMBER_TWEETS - 1)) {
-            tweets[numTweets] = keyboard.nextLine();
+            tweets[numTweets] = keyboard.nextLine() + " " + getCurrentTimeStamp();
             numTweets++;
             
             System.out.println(tweeterName + "'s Personal Twitter Feed:");
             for(int i = 0; i < numTweets; i++) {
                 System.out.println("- " + tweets[i]);
```

Comment:
I found it easier and added the TimeStamp while in the loop after printing each tweet with tweets[i] + Data();